addons/l10n_eu_oss/models/res_company.py

```diff
@@ -136,7 +140,8 @@ def _get_oss_account(self):
 
     def _get_oss_tags(self):
         oss_tag = self.env.ref('l10n_eu_oss.tag_oss')
-        tag_for_country = EU_TAG_MAP.get(self.chart_template, {
+        chart_template = self.env['account.chart.template']._guess_chart_template(self.account_fiscal_country_id)
```

Comment:
```suggestion
        country = None
        # Try to use the VAT country if easily guessable
        country_prefix = re.match('^[a-zA-Z]{2}|^', self.vat).group()
        if country_prefix:
            country = self.env['res.country'].search([('code', '=', country_prefix)], limit=1)
        # otherwise fallback on the fiscal country
        if not country:
            country = self.account_fiscal_country_id
        chart_template = self.env['account.chart.template']._guess_chart_template(country)
```
Didn't test yet, naive approach to cover the need as asked by tsb (use the VAT to guess the country)